roles/ceph-mds/tasks/create_mds_filesystems.yml

```diff
@@ -2,7 +2,17 @@
 - name: filesystem pools related tasks
   block:
     - name: create filesystem pools
-      command: "{{ hostvars[groups[mon_group_name][0]]['docker_exec_cmd'] | default('') }} ceph --cluster {{ cluster }} osd pool create {{ item.name }} {{ item.pgs | default(osd_pool_default_pg_num) }}"
+      command: >
+        {{ hostvars[groups[mon_group_name][0]]['docker_exec_cmd'] | default('') }} ceph --cluster {{ cluster }}
+        osd pool create {{ item.name }}
+        {{ item.pg_num | default(item.pgs) | default(osd_pool_default_pg_num) }}
+        {{ item.pgp_num | default(item.pgs) | default(item.pg_num) | default(osd_pool_default_pg_num) }}
+        {{ 'replicated_rule' if not item.rule_name | default('replicated_rule') else item.rule_name | default('replicated_rule') }}
```

Comment:
NOTE for @dsavineau :

If I remember correctly, I think this complexity was added to keep backward compatibility in previous releases.
Since this wasn't introduced by this patch, I think it's fine to merge this anyway but we should consider and check if it is possible to deprecate and remove this in a dedicated PR for the next release.

Reply:
agree